apps/explorer/priv/repo/migrations/20220928075913_add_aggregation_index_to_celo_election_rewards.exs

```diff
@@ -0,0 +1,7 @@
+defmodule Explorer.Repo.Migrations.AddAggregationIndexToCeloElectionRewards do
+  use Ecto.Migration
+
+  def change do
+    create(index(:celo_election_rewards, [:block_number, :amount, :reward_type]))
```

Comment:
Why do you need to include `amount` into the index?

Reply:
because we filter by non-zero amount: https://github.com/celo-org/blockscout/pull/761/files#diff-bae11f52aabd1b6e9a6b9924a15b0673dbba237b0bf18b013aaaebe5d7ebb153R255

Reply:
Ah, makes sense.